.github/workflows/build_test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.5, 3.6]
+        python-version: [3.6]
```

Comment:
Could we add 3.7 to the list, now that 3.6 is EOL?

Reply:
I added the version.